r-package/grf/R/deprecated.R

```diff
@@ -9,7 +9,7 @@
 #'
 #' @export
 average_late <- function(forest, ...) {
-  stop("This function has been deprecated after version 1.2.0. See the function `average_treatment_effect` instead.")
+  stop("This function has been removed after version 1.2.0. See the function `average_treatment_effect` instead.")
```

Comment:
Maybe we could update the title in the comment to "Average LATE (removed)"

Reply:
Oki